hazelcast/src/main/java/com/hazelcast/client/config/ClientFailoverConfig.java

```diff
@@ -31,18 +31,32 @@
 import static com.hazelcast.client.config.ClientConnectionStrategyConfig.ReconnectMode.OFF;
 
 /**
- * Config class to configure multiple client configs to be used by single client instance
- * The client will try to connect them in given order.
- * When the connected cluster fails or the client blacklisted from the cluster via the management center, the client will
+ * Config class to configure multiple client configs to be used by single client instance.
+ * The client will try to connect them in given order. When the connected cluster fails
+ * or the client blacklisted from the cluster via the Management Center, the client will
  * search for alternative clusters with given configs.
+ * <p>
+ * The client configurations must be exactly the same except the following configuration options:
```

Comment:
Looking at the implementation, it seems that `clusterName` also does not have to match

Reply:
Yes, you're right. I took the list from the ref manual. cc @Serdaro

Reply:
Addressed in ce976c31e4e9bd3a6920d42540a62fcdd12b1f83